packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -615,6 +615,59 @@ function getRank(
   return getRankOrder(memberGroups, orderConfig);
 }
 
+/**
+ * If there are more than 1 consecutive same ranked member in 1 class, those members are put in an array & an array of these members' array is returned as a result. If, for example, the memberSet parameter looks like the below example - (contd. after the example tag)
+ * @example
+ * ```
+ * interface Foo {
+      [a: string]: number;
+ 
+      a: x;
+      B: x;
+      c: x;
+ 
+      c(): void;
+      B(): void;
+      a(): void;
+ 
+      (): Baz;
+ 
+      new (): Bar;
+    }
+ * ```
+  The resulting array will look like this - [[a, B, c], [c, B, a]].
+ * @param memberSet The members in the class of the source code on which the grouping operation will be performed.
+ * @param memberType The member type order listed in the eslint config file.
+ * @param supportsModifiers it'll get passed to getRank()
+ * @returns the array of groups of members
+ */
+function groupMembersByType(
+  memberSet: Member[],
+  memberType: MemberType[],
```

Comment:
[Naming] And for this one, it's passed in as `memberTypes: MemberType[]` (plural) but here is referred to as `memberType: MemberType[]` (singular). Is there a reason to change?

If not:

```suggestion
  memberTypes: MemberType[],
```